src/interpreter/kernel/exec.js

```diff
@@ -15,11 +15,13 @@ const i64 = require("../runtime/values/i64");
 const f32 = require("../runtime/values/f32");
 const f64 = require("../runtime/values/f64");
 const label = require("../runtime/values/label");
-const { createChildStackFrame } = require("./stackframe");
+const stackframe = require("./stackframe");
 const { createTrap } = require("./signals");
 const { RuntimeError } = require("../../errors");
 const t = require("../../compiler/AST");
 
+// DEFINE_MACRO(CHECK_STACK_UNWIND, 'if (frame._unwindReason !== 0) { break; }');
```

Comment:
I'm trying to get a macro system working, I'm looking for a c-like macro system.
The existing one in Babel doesn't work since Babel can't parse the illegal `break` statement here "lol".